runtime/centrifuge/src/lib.rs

```diff
@@ -2220,6 +2220,16 @@ impl_runtime_apis! {
 			let pool = pallet_pool_system::Pool::<Runtime>::get(pool_id)?;
 			pool.tranches.tranche_currency(tranche_loc).map(Into::into)
 		}
+
+		fn nav(pool_id: PoolId) -> Option<(Balance, pallet_pool_system::Nav<Balance>)> {
+			let pool = pallet_pool_system::Pool::<Runtime>::get(pool_id)?;
+			let nav_loans = Loans::update_nav(pool_id).ok()?;
```

Comment:
Just to be sure about this. Do we want the NAV value used to compute the loan's portfolio, or is the NAV value just for the moment we call the RuntimeAPI?

Reply:
I was not sure and asked @mustermeiszer [who said that we want the current NAV](https://kflabs.slack.com/archives/C04H9H7EVEE/p1706279505349109?thread_ts=1706028509.406749&cid=C04H9H7EVEE), even if the onchain one is outdated.

Reply:
Thanks to share the double check!

Reply:
Definitely current one, as the UI needs to have the latest data!